### ISSUE TITLE
new feature to add a Notes field mask option

### ISSUE THREAD

**Issue by skuldouggery** [PhilippC/keepass2android#182]

**Comment by 10101000** [comment]
Yes please add this. Some of us use the notes field to save password reset questions and backup 2-step codes. Thanks for this great application. It's very much worth the donation!

**Comment by PhilippC** [comment]
please keep in mind that Keepass2Android is just one app to use Keepass databases. Other apps won't consider this setting. It's more standards compliant to add a new protected field to the entry.
